bin/modules-docs.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var https = require('https');
+var logger = require('console');
+
+var docs = path.join(__dirname, '..', 'docs', 'modules');
+var moduleReadmes = {
+    'geval': ['Raynos/geval', '2.1.1', 'README.md'],
```

Comment:
If you store the version numbers here you'll have to manually or programmatically update this file. both sound like a bad idea. Instead grab the version from the package.json. Also, why are we downloading from github when they should already be in our node_modules folder?